documentation/book/proc-accessing-kafka-using-loadbalancers.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// assembly-configuring-kafka-listeners.adoc
+
+[id='proc-accessing-kafka-using-loadbalancers-{context}']
+= Accessing Kafka using loadbalancers routes
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Deploy Kafka cluster with external listener enabled and configured to the type `loadbalancer`.
```

Comment:
1. Deploy a Kafka cluster with an external listener enabled and configured with the type `loadbalancer`.